firestore-tos-extension/PREINSTALL.md

```diff
@@ -0,0 +1,82 @@
+Use this extension to provides an API for adding new ackowledgement notices, retrieving the latest notices, and tracking acknowledgements.
+
+Allows users to sign ackowledgement forms as part of their application. Some common ackowledgement forms include notice and cookie ackowledgement.
+
+## Usage
+
+The extension supports multiple use cases:
+
+- Accepting notices
+- Creating notices
+- Getting notices
+- Getting accepted notices
+
+All functions `require authentication` to run, ensure you have a valud Firebase User included in your POST requests.
+
+Managed documents are stored in a specified Cloud Firestore collection, while providing two subcollections for accesing data, these are:
+
+- acknowledgements
+- agreements
+
+Each collection has a managed section for supporting multiple instances of the extension. For example, a typical routes for agreements would be:
```

Comment:
Let's get agreement out of here... find another example?

Reply:
Same as https://github.com/FirebaseExtended/experimental-extensions/pull/91#discussion_r964961203

Reply:
Updated with new config updates.